code/go/0chain.net/chaincore/chain/entity.go

```diff
@@ -611,6 +618,69 @@ func (c *Chain) setupInitialState(initStates *state.InitStates) util.MerklePatri
 	return pmt
 }
 
+func (c *Chain) addInitialStakes(mpt *util.MerklePatriciaTrie, stakes []state.InitStake) error {
+	var edbDelegatePools []event.DelegatePool
+	for _, v := range stakes {
+		key := v.ProviderType + ":stakepool:" + v.ProviderID
+		sp := stakepool.StakePool{}
+		if err := mpt.GetNodeValue(util.Path(encryption.Hash(key)), &sp); err != nil {
+			if err != util.ErrValueNotPresent {
+				continue
+			}
+			sp.Pools = map[string]*stakepool.DelegatePool{}
+		}
+
+		_, ok := sp.Pools[v.ClientID]
+		if ok {
+			continue
+		}
+
+		sp.Pools[v.ClientID] = &stakepool.DelegatePool{
+			Balance:      v.Tokens,
+			Reward:       0,
+			Status:       0,
```

Comment:
We do not need to set zero values explicitly

Reply:
Sure, I can remove them.